src/Utils/Miscellaneous.php

```diff
@@ -20,4 +20,55 @@ public static function systemTime()
     {
         return intval(round(microtime(true) * 1000));
     }
+
+    private static function deepConvertObjectToArrayRecursive(&$item , $_key)
```

Comment:
can we add a separate test for `deepConvertObjectToArrayRecursive` under `MiscellaneousTest.php`